internal/stokerhttp/get.go

```diff
@@ -1,48 +1,66 @@
 package stokerhttp
 
 import (
-	"context"
 	"errors"
 	"fmt"
 	"net/http"
-	"net/url"
 	"strconv"
 	"time"
 
-	"github.com/frantjc/sindri/internal/appinfoutil"
 	"github.com/frantjc/sindri/steamapp/postgres"
 	"github.com/go-chi/chi"
 	"github.com/go-logr/logr"
 )
 
-type SteamappMetadata struct {
-	AppID       int       `json:"app_id"`
-	Name        string    `json:"name,omitempty"`
-	IconURL     string    `json:"icon_url,omitempty"`
-	DateCreated time.Time `json:"date_created"`
-	DateUpdated time.Time `json:"date_updated"`
-	Locked      bool      `json:"locked,omitempty"`
+type SteamappList struct {
+	Offset    int        `json:"offset"`
+	Limit     int        `json:"limit"`
+	Steamapps []Steamapp `json:"steamapps"`
+}
+
+type Steamapp struct {
+	SteamappSpec
+	SteamappInfo
 }
 
 type SteamappSpec struct {
```

Comment:
I don't like that `SteamappSpec`, `SteamappInfo`, and their `from` functions are defined in `get.go` but are used in both `get.go` and `upsert.go`. I think we should move shared models to a different file